zone/client_process.cpp

```diff
@@ -193,6 +193,10 @@ bool Client::Process() {
 			return false; //delete client
 		}
 
+		if (bot_camp_timer.Check()) {
```

Comment:
Should check if bots are enabled otherwise we waste cycle needlessly for servers that don't have bots enabled